tests/test_circulationapi.py

```diff
@@ -898,27 +898,33 @@ def yes_we_can(*args, **kwargs):
         result = try_to_fulfill()
         eq_(fulfillment, result)
 
-    def test_revoke_loan_sends_analytics_event(self):
+    def test_revoke_loan(self):
+        self.patron.last_loan_activity_sync = datetime.utcnow()
         self.pool.loan_to(self.patron)
         self.remote.queue_checkin(True)
 
         result = self.circulation.revoke_loan(self.patron, '1234', self.pool)
-
         eq_(True, result)
 
+        # The patron's loan activity is now out of sync.
+        eq_(None, self.patron.last_loan_activity_sync)
+
         # An analytics event was created.
         eq_(1, self.analytics.count)
         eq_(CirculationEvent.CM_CHECKIN,
             self.analytics.event_type)
 
-    def test_release_hold_sends_analytics_event(self):
+    def test_release_hold(self):
+        self.patron.last_loan_activity_sync = datetime.utcnow()
```

Comment:
Using `utcnow()` here could cause the test to fail if, in the future, `loan_activity_max_age` is set to a very small positive value or zero.